test/test.js

```diff
@@ -376,6 +376,36 @@ suite('Response', function() {
       return json;
     })
   })
+  test('clone text response', function() {
+    var r = new Response('{"foo":"bar"}', {headers: {'content-type': 'application/json'}}),
+        clone = r.clone();
+    assert(clone.headers !== r.headers, true);
```

Comment:
There's no need to pass `true` as 2nd argument to `assert`. The 2nd argument is optional message in case of failure